R/derive_advs_params.R

```diff
@@ -234,7 +234,7 @@ compute_map <- function(diabp, sysbp, hr = NULL) {
 #'   parameter) and to the parameters specified by `HEIGHT` and `WEIGHT`.
 #'
 #' @param method Derivation method to use. Note that `HEIGHT` is expected
-#'    in cm and `WEIGHT` is expected in kg:
+#'    in m and `WEIGHT` is expected in kg:
```

Comment:
Height is still expected in cm. Please undo this change here and below.

Reply:
> Height is still expected in cm. Please undo this change here and below.
@bundfussr - my error!   Will undo and correct (to `cm`)